db/schema.rb

```diff
@@ -10,11 +10,34 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170116121348) do
+ActiveRecord::Schema.define(version: 20170427164809) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "actions", force: :cascade do |t|
+    t.integer  "workflow_id",               null: false
+    t.integer  "subject_id",                null: false
+    t.string   "effect_type",               null: false
+    t.jsonb    "config",       default: {}, null: false
+    t.integer  "status",       default: 0,  null: false
+    t.datetime "attempted_at"
+    t.datetime "completed_at"
+    t.datetime "created_at",                null: false
+    t.datetime "updated_at",                null: false
+    t.index ["subject_id"], name: "index_actions_on_subject_id", using: :btree
+    t.index ["workflow_id"], name: "index_actions_on_workflow_id", using: :btree
+  end
+
+  create_table "classifications", force: :cascade do |t|
+    t.jsonb    "annotations",              array: true
+    t.integer  "project_id"
+    t.integer  "workflow_id"
+    t.integer  "user_id"
+    t.datetime "inserted_at", null: false
+    t.datetime "updated_at",  null: false
+  end
```

Comment:
Hey, nice catch. Don't know why this was included in this PR. I'll rebase it out.